cypress/integration/06.WritePost.feature

```diff
@@ -17,7 +17,7 @@ Feature: Create a post
     for active citizenship.
     """
     And I click on "Save"
-    Then I get redirected to "/post/my-first-post/"
+    Then I get redirected to ".../my-first-post"
```

Comment:
It's not so clear how this works until one checks out https://github.com/Human-Connection/Human-Connection/pull/248/files#diff-cc51d55abc8d6a1b33fd1f8885365953R227, but maybe there is nothing one can do about that